vnext/ReactUWP/Modules/Animated/SpringAnimationDriver.h

```diff
@@ -16,44 +16,45 @@ class SpringAnimationDriver : public AnimationDriver {
       int64_t animatedValueTag,
       const Callback &endCallback,
       const folly::dynamic &config,
-      const std::shared_ptr<NativeAnimatedNodeManager> &manager);
+      const std::shared_ptr<NativeAnimatedNodeManager> &manager,
+      const folly::dynamic &dynamicToValues = folly::dynamic::array());
 
-  std::tuple<winrt::CompositionAnimation, winrt::CompositionScopedBatch>
-  MakeAnimation(const folly::dynamic &config) override;
+std::tuple<winrt::CompositionAnimation, winrt::CompositionScopedBatch>
+MakeAnimation(const folly::dynamic &config) override;
 
- private:
-  std::tuple<float, double> GetValueAndVelocityForTime(
-      double time,
-      double startValue);
-  bool IsAtRest(
-      double currentVelocity,
-      double currentPosition,
-      double startPosition);
-  bool IsOvershooting(double currentValue, double startValue);
+double ToValue() override;
 
-  double m_springStiffness{0};
-  double m_springDamping{0};
-  double m_springMass{0};
-  double m_initialVelocity{0};
-  double m_endValue{0};
-  double m_restSpeedThreshold{0};
-  double m_displacementFromRestThreshold{0};
-  bool m_overshootClampingEnabled{0};
-  int m_iterations{0};
+private:
+std::tuple<float, double> GetValueAndVelocityForTime(
+    double time,
+    double startValue);
+bool IsAtRest(double currentVelocity, double currentPosition, double endValue);
+bool IsOvershooting(double currentValue, double startValue);
 
-  static constexpr std::string_view s_springStiffnessParameterName{"stiffness"};
-  static constexpr std::string_view s_springDampingParameterName{"damping"};
-  static constexpr std::string_view s_springMassParameterName{"mass"};
-  static constexpr std::string_view s_initialVelocityParameterName{
-      "initialVelocity"};
-  static constexpr std::string_view s_endValueParameterName{"toValue"};
-  static constexpr std::string_view s_restSpeedThresholdParameterName{
-      "restSpeedThreshold"};
-  static constexpr std::string_view
-      s_displacementFromRestThresholdParameterName{"restDisplacementThreshold"};
-  static constexpr std::string_view s_overshootClampingEnabledParameterName{
-      "overshootClamping"};
-  static constexpr std::string_view s_iterationsParameterName{"iterations"};
-};
-} // namespace uwp
+double m_springStiffness{0};
+double m_springDamping{0};
+double m_springMass{0};
+double m_initialVelocity{0};
+double m_endValue{0};
+double m_restSpeedThreshold{0};
+double m_displacementFromRestThreshold{0};
+bool m_overshootClampingEnabled{0};
+int m_iterations{0};
+folly::dynamic m_dynamicToValues{};
+
+static constexpr std::string_view s_springStiffnessParameterName{"stiffness"};
+static constexpr std::string_view s_springDampingParameterName{"damping"};
+static constexpr std::string_view s_springMassParameterName{"mass"};
+static constexpr std::string_view s_initialVelocityParameterName{
+    "initialVelocity"};
```

Comment:
I think pretty universally everyone thinks the ColumnLimit of 80 in the .clang-format is too low. We snapped someone else's as a starting point. If you want to take a PR to bump that to even just 100 I think you'd be welcomed with open arms :)